pylatex/document.py

```diff
@@ -197,19 +199,45 @@ def generate_pdf(self, filepath=None, *, clean=True, clean_tex=True,
         if compiler_args is None:
             compiler_args = []
 
-        filepath = self._select_filepath(filepath)
-        filepath = os.path.join('.', filepath)
+        python_version = '{}.{}.{}'.format(sys.version_info[0],
+                                           sys.version_info[1],
+                                           sys.version_info[2])
+        python_cwd_available = False
+
+        if version.parse(python_version) > version.parse("3.6"):
```

Comment:
You can change this for:
```suggestion
        if sys.version_info >= (3, 6):
```

Then you can stop importing `packaging` too.

Reply:
done